artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ServerSessionImpl.java

```diff
@@ -1259,6 +1259,7 @@ public RoutingStatus send(final ServerMessage message, final boolean direct, boo
       return send(getCurrentTransaction(), message, direct, noAutoCreateQueue);
    }
 
+   @Override
```

Comment:
@clebertsuconic if you have your Error Prone changes still somewhere, maybe you can push them somewhere public, I'd be interested in taking a look?